src/pineko/theory.py

```diff
@@ -15,11 +15,21 @@
 import rich
 import yaml
 
-from . import check, configs, evolve, parser, theory_card
+from . import check, configs, evolve, parser, scale_variations, theory_card
 
 logger = logging.getLogger(__name__)
 
 
+def check_scvar_evolve(grid, max_as, max_al, tocheck):
+    """Check if the scale variations and central orders are to enough to construct the fktable."""
```

Comment:
```suggestion
    """Check scale variations presence and consistency.
    
    In particular, check if the scale variations and central orders are those 	
    required to construct a scale-varied fktable.
    
    """
```

Try to keep the first line as brief as possible (but meaningful)

Reply:
[bc2c141](https://github.com/NNPDF/pineko/pull/75/commits/bc2c14168987895dd9926d2f2743034829e9d5f6)